pyvrp/tests/test_ProblemData.py

```diff
@@ -66,46 +65,10 @@ def test_depot_is_first_client():
 
     data = ProblemData(
         clients=[Client(x=0, y=0), Client(x=0, y=1)],
-        nb_vehicles=1,
+        num_vehicles=1,
         vehicle_cap=1,
         distance_matrix=mat,
         duration_matrix=mat,
     )
 
     assert_(data.depot() is data.client(0))
-
-
-def test_matrix_access():
-    """
-    Tests that the ``duration()``, ``duration_matrix()``, ``dist()``, and
-    ``distance_matrix()`` methods correctly index the underlying duration
-    and distance matrices.
-    """
-    gen = default_rng(seed=42)
-    size = 6
-
-    dist_mat = gen.integers(500, size=(size, size))
-    dur_mat = gen.integers(500, size=(size, size))
-    clients = [
-        Client(x=0, y=0, demand=0, service_duration=0, tw_early=0, tw_late=10)
-        for _ in range(size)
-    ]
-
-    data = ProblemData(
-        clients=clients,
-        nb_vehicles=1,
-        vehicle_cap=1,
-        distance_matrix=dist_mat,  # type: ignore
-        duration_matrix=dur_mat,  # type: ignore
-    )
-
-    dist_mat_data = data.distance_matrix()
-    dur_mat_data = data.duration_matrix()
-
-    for frm in range(size):
-        for to in range(size):
-            assert_allclose(dur_mat[frm, to], data.duration(frm, to))
-            assert_allclose(dist_mat[frm, to], data.dist(frm, to))
-
-            assert_allclose(dur_mat_data[frm, to], dur_mat[frm, to])
-            assert_allclose(dist_mat_data[frm, to], dist_mat[frm, to])
```

Comment:
Why remove this test? If it is because we no longer bind the matrices in problem data, the test is still useful internally?